Gems/LmbrCentral/Code/include/LmbrCentral/Component/EditorWrappedComponentBase.inl

```diff
@@ -222,7 +222,12 @@ namespace LmbrCentral
     template <typename TComponent, typename TConfiguration>
     AZ::u32 EditorWrappedComponentBase<TComponent, TConfiguration>::ConfigurationChanged()
     {
-        m_component.Deactivate();
+        // Only call Deactivate if it's currently active.
+        if (m_component.GetEntity())
+        {
+            m_component.Deactivate();
+        }
+
```

Comment:
This is confusing. If `m_component.GetEntity()` tests if the component is active, then why do we check that same thing _before_ calling `Activate()` on it? Also since m_component is just a TComponent, we have no way of knowing if the component has overridden the default behavior of `GetEntity()`

Reply:
I'll back this change out of the PR. It was reducing the number of extra Deactivate() calls that I was seeing, but in trying to get answers to your questions, I verified that it doesn't fully eliminate them. I'll hold off on this until I can find a more complete change.

Reply:
Actually, put the change back in, but this time by adding more comprehensive state tracking of the nested wrapped component.  It no longer relies on GetEntity().